test/unit/helpers/ast-node-info-test.js

```diff
@@ -0,0 +1,13 @@
+var assert = require('power-assert');
+var parse = require('htmlbars/dist/cjs/htmlbars-syntax').parse;
+var AstNodeInfo = require('../../../lib/helpers/ast-node-info');
+
+describe('isImgElement', function() {
+  it('can detect an image tag', function() {
+    var tableAst = parse('<table></table>');
+    assert(AstNodeInfo.isImgElement(tableAst.body[0]) === false);
+
+    var imgAst = parse('<img />');
```

Comment:
@Robdel12 - Nope, definitely doesn't throw.  But then if that were the only reason to change behaviors this linter wouldn't exist 😉 .